nixos/modules/misc/ids.nix

```diff
@@ -288,6 +288,7 @@
       kresd = 270;
       rpc = 271;
       geoip = 272;
+      slimserver = 273;
```

Comment:
It should probably be made clearer why one might want to use static ids, at any rate; it's probably heavily overused as-is, leading to the impression that it's required.